main_logic/tts_client.py

```
@@ -1030,3 +1044,157 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
+
+                # 重连后重启接收任务
+                if receive_task and not receive_task.done():
+                    receive_task.cancel()
+                receive_task = asyncio.create_task(receive_loop(ws, resampler, response_queue))
+                return ws
+            except Exception as e:
+                logger.error(f"连接失败: {e}")
+                ws = None
+                return None
+
+        # 1. 初始连接 (先连上再发 ready 信号，防止死锁)
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)